### PR TITLE
Changed start/end selecting to slider bar and fixed label position.

### PR DESCRIPTION
**Pull request by asunawu**
Fixed [AB#1757](https://dev.azure.com/euclideon/57c3bd2a-8f94-4578-b195-2e4fa9d8f295/_workitems/edit/1757).

Changed start/end selecting to slider bar and fixed label position.

### CODE REVIEW

File: src/scene/vcVerticalMeasureTool.cpp [hunk 1]
```diff
@@ -64,8 +63,7 @@ void vcVerticalMeasureTool::EndMeasure(vcState *pProgramState, const udDouble3 &
 
 void vcVerticalMeasureTool::OnNodeUpdate(vcState *pProgramState)
 {
-  vdkProjectNode_GetMetadataBool(m_pNode, "pickStart", &m_pickStart, true);
-  vdkProjectNode_GetMetadataBool(m_pNode, "pickEnd", &m_pickEnd, true);
+  vdkProjectNode_GetMetadataInt(m_pNode, "selectedPoint", &m_selectedPoint, -1);
```
Comment (samsinsane) [inline src/scene/vcVerticalMeasureTool.cpp:66]: @OsiJr Should we be handling the old metadata going forward?

Comment (OsiJr) [reply src/scene/vcVerticalMeasureTool.cpp:66]: Yes but why are these attributes even being saved to the project?

Comment (asunawu) [reply src/scene/vcVerticalMeasureTool.cpp:66]: Sorry I thought we needed to save all attributes, will remove it.

Comment (asunawu) [reply src/scene/vcVerticalMeasureTool.cpp]: Removed.

**Pull request merged**